docs/database-access/accessing-the-developer-workspace-and-adding-a-database-instance/quickstart.yml

```diff
@@ -14,15 +15,15 @@ spec:
   description: 'A quick step-by-step guide for developers using OpenShift Database Access to access the developer workspace, and connect database instances from cloud-database providers to their applications.'
-  prerequisites: ["Access to the OpenShift console.", "Creation of a provider account resource."]
+  prerequisites: ["Access to the OpenShift console.", "Installation of the Red Hat OpenShift Database Access add-on.", "Importing at least one cloud-database provider account."]
```

Comment:
For consistency with the other items in the list, should it be "Import of ..." instead of "Importing ..."?
It would be even better to use complete sentences for each prerequisite, rather than sentence fragments.

This same comment applies to the docs/database-access/connecting-an-application-to-a-database-instance-using-the-topology-view/quickstart.yml file (for some reason, I can't add a comment there).

Reply:
Thanks for the feedback!  I've updated the source accordingly.